generated/nidaqmx/_base_interpreter.py

```diff
@@ -0,0 +1,1574 @@
+# Do not edit this file; it was automatically generated.
+from typing import List
+
+class BaseInterpreter(object):
+    """
+    Contains signature of functions for all DAQmx APIs.
+    """
+    def add_cdaq_sync_connection(self, port_list):
+        pass
```

Comment:
If someone imports this class and decides to write their on `Interpreter` class (or we in future decide to add another one), they should implement all of these method, right?

To document this, rather than putting "pass", have these default implementations do a `raise NotImplemented()`, or (if using abc.ABC, mark them as [`@abstractmethod`](https://docs.python.org/3.9/library/abc.html#abc.abstractmethod)